benchmarks/DiscoveryBenchmark.php

```
@@ -68,13 +66,9 @@ public function contextualize(ContextInterface $context): void {
         }
 
       };
-
-      Environment::addProvider($provider);
     }
 
-    // Access providers to ensure they are loaded.
-    Environment::provider();
-    Environment::type();
+    Environment::init(providers: $providers);
```

PHPMD (2.15.0)

Avoid using static access to class '\DrevOps\EnvironmentDetector\Environment' in method 'benchAddProvider'. (undefined)

(StaticAccess)